api/api/auth/tests.py

```diff
@@ -138,6 +139,51 @@ def test_createToken_tokenExists_returnsExistingTokenWithUserInfo(
                          f'Got {users_count} users in db, expected 1')
         self.assertEqual(response.data['token'], existing_token.token)
 
+    @pytest.mark.django_db
+    @mock.patch('time.sleep')
+    @mock.patch('google_auth_oauthlib.flow.Flow.fetch_token')
+    @mock.patch('google.oauth2.id_token.verify_oauth2_token')
+    def test_createExpiredToken_tokenExists_returnsNewTokenWithUserInfo(
```

Comment:
Test should not describe the implementation detail of the test.
just say, `expiredTokenExists` instead of `createExpiredToken`.
When somebody reads a test, they want to know what the test does instead of how it does it.